tox.ini

```
@@ -60,7 +60,6 @@ deps =
     -r dev_requirements/requirements-type_check.txt
 commands =
     mypy --show-error-codes tests
```

```suggestion
    mypy --show-error-codes tests
    mypy src/borm
```

oder ggf auch ohne src. schau mal im template repo

---

9956d9dff41582e0020b6b5e890595773dad4554